metaflow/plugins/argo/argo_live_run.py

```diff
@@ -154,22 +161,19 @@ def trigger(
                 loop_counter += 1
                 time.sleep(5)
             else:
-                raise TimeoutError(f"Failed to begin running within {wait_timeout} minutes")
+                raise TimeoutError(
+                    f"Failed to begin running within {wait_timeout} minutes"
+                )
 
             success_statement = (
-                "successfully!!!" if self.successful else "unsuccessfully."
+                "successfully!!!" if live_run.successful else "unsuccessfully."
```

Comment:
Not necessarily in the same PR, but can wait be extracted as a separate function?

Reply:
I created a new JIRA task [AIP-6554](https://zbrt.atl.zillow.net/browse/AIP-6554) for this comment.